Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Security - Simple Passwords Disabled.json

```diff
@@ -12,7 +12,7 @@
       }
     }
   ],
-  "description": "Block simple passwords including picture passwords as they don't meet corporate device password policy.",
+  "description": "What is it: Blocks simple passwords, including picture passwords. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-password-ban-bad",
```

Comment:
**What does this do?**
Blocks simple passwords including picture passwords.

**Why should you use this?**
If you want to increase security by blocking simple passwords. 

**What is the end-user impact?**
Users are restricted from creating simple passwords. 

**Learn more**
[Eliminate bad passwords using Azure Active Directory Password Protection](https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-password-ban-bad)